### PR TITLE
Docs index

### PR DESCRIPTION
**Pull request by andrewdotn**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -6,13 +6,45 @@
...
+  descriptions.
+
 .. toctree::
    :maxdepth: 2
    :caption: Contents:
 
-   paradigm-layouts
-   ci
-   developers-guide
+   # TOOD: add conceptual
```
Comment (dwhieb) [inline docs/index.rst:42]:
```suggestion
   # TODO: add conceptual
```

A comment here doesn't mess up the list of files to include in toctree?

**Review by dwhieb — APPROVED** [review]
One tiny question, otherwise looks good! I'll follow up with more revisions in another PR soon.

File: docs/glossary.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ to describing certain languages.
 
 ## Terms
 
+<!-- https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#syntax-directives -->
```
Comment (dwhieb) [inline docs/glossary.md:10]: 👍🏻

File: docs/index.rst (hunk 1 above)
Comment (andrewdotn) [reply docs/index.rst]: `toctree` seems to ignore it but you’re right it’s not valid syntax, so I’ve removed it

**Pull request merged**